### ISSUE TITLE
[BUG] Hard-coded value of year in the copyright section.

### ISSUE THREAD

**Issue by abhinav-m22** [WebXDAO/WebXDAO.github.io#361]
### Description of the bug

In the copyright section of the web page, the year 2023 is hard-coded.
It is not the standard way of writing code.
Everytime the code will have to be changed for updating the year.
Instead, add a function to dynamically fetch the year from a function which is the best practice to add the copyright to any website to make it look formal.

### Please add screenshots (if applicable)

<img width="675" alt="image" src="https://github.com/WebXDAO/WebXDAO.github.io/assets/113239388/57ab2b7d-2c7d-41bd-b78f-ff0a3640d221">

<img width="539" alt="image" src="https://github.com/WebXDAO/WebXDAO.github.io/assets/113239388/67b54206-ae6c-43ea-9b29-ccbf176ac042">

### Add any other context about the problem here

I want to work on this feature implementation.
Please assign me task under GSSoC'23.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by abhinav-m22** [comment]
> Hello! Nice catch. Do you want to be assigned?

Yes please.Or else someone could work on the same feature simultaneously which will be an issue.

**Comment by mkubdev** [comment]
Footer is merged now. So we assign you! :D
